app/components/atoms/SearchBar/SearchBar.js

```diff
@@ -1,14 +1,23 @@
 import React, { Component } from 'react';
 
+import { create } from 'apisauce';
+
 import { SearchBar } from '@rneui/themed';
 
+
 const style = {
   container: {
     flex: 1,
     backgroundColor: 'powderblue'
   }
 };
 
+const api = create({
+  baseURL: 'https://itunes.apple.com/',
+})
+
+const getItunesItem = () => api.get('search?term=perfect')
```

Comment:
Please refer [here](https://github.com/wednesday-solutions/react-native-template/tree/master/app/services) to go through how we define a particular service method.

lets define it there and import it into this file

it wil be like 
```
// @services/Itunes.js

const api = ...

export const getItunes = () => api.get(..); 

// and then inside @atoms/SearchBar/index.js
import { getItunes } from `@services/itunes`

```